packages/docusaurus-plugin-sitemap/src/pluginOptionSchema.ts

```diff
@@ -16,7 +17,7 @@ export const DEFAULT_OPTIONS: Required<PluginOptions> = {
 };
 
 export const PluginOptionSchema = Joi.object({
-  cacheTime: Joi.number().positive().default(DEFAULT_OPTIONS.cacheTime),
+  cacheTime: Joi.number().positive().default(DEFAULT_OPTIONS.cacheTime), // deprecated
```

Comment:
How do we manage deprecated fields? Should we notify the user about this (via `console.log`)?

Reply:
We have some examples in classic theme:

```
  disableDarkMode: Joi.any().forbidden(false).messages({
    'any.unknown':
      'disableDarkMode theme config is deprecated. Please use the new colorMode attribute. You likely want: config.themeConfig.colorMode.disableSwitch = true',
  }),
```

(not sure why there's a "false", looks like a typo btw)